kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1445,30 +1461,30 @@ Precompiled Contracts
          <output> _ => Int2Bytes(32, 4096, BE) +Bytes Int2Bytes(32, blsModulus, BE) </output>
          <callData> CD </callData>
       requires lengthBytes( CD ) ==Int 192
-       andBool #kzg2vh(substrBytes(CD, 96, 144)) ==K substrBytes(CD, 0, 32)
+       andBool #kzg2vh(substrBytes(CD, 96, 144)) ==Bytes substrBytes(CD, 0, 32)
        andBool Bytes2Int(substrBytes(CD, 32, 64), BE, Unsigned) <Int blsModulus
        andBool Bytes2Int(substrBytes(CD, 64, 96), BE, Unsigned) <Int blsModulus
        andBool verifyKZGProof(substrBytes(CD, 96, 144), substrBytes(CD, 32, 64), substrBytes(CD, 64, 96), substrBytes(CD, 144, 192))
 
     rule <k> KZGPOINTEVAL => #end EVMC_PRECOMPILE_FAILURE ... </k>
-         <callData> CD </callData>
```

Comment:
You can't delete this cell, as it's used by the `requires` clause: See the CI error here https://github.com/Pi-Squared-Inc/revm/actions/runs/15536273144/job/43735940632?pr=53

Reply:
Aah, this is a typo, caused by the rebase of the PR.